blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/BluefloodServiceStarter.java

```diff
@@ -313,4 +316,20 @@ public void run() {
         startEventListenerModules();
         log.info("All blueflood services started");
     }
+
+    @VisibleForTesting
+    public static boolean SkipInstantiateRestartGauge = false;
+
+    @VisibleForTesting
+    public static boolean ThrowInsteadOfExit = false;
```

Comment:
I am not a big fan of test code cluttering main code. Is it possible to just mock the System.exit() code?
http://stackoverflow.com/questions/309396/java-how-to-test-methods-that-call-system-exit